tests/serial/convergence-measures/testConvergenceMeasures1.xml

```diff
@@ -45,7 +45,6 @@
     <exchange data="Data1" mesh="MeshOne" from="SolverOne" to="SolverTwo" />
     <exchange data="Data2" mesh="MeshOne" from="SolverTwo" to="SolverOne" />
     <max-iterations value="100" />
-    <absolute-convergence-measure limit="1e-1" data="Data2" mesh="MeshOne" suffices="true" />
-    <min-iteration-convergence-measure min-iterations="3" data="Data1" mesh="MeshOne" />
+    <absolute-convergence-measure limit="1e-1" data="Data2" mesh="MeshOne" />
```

Comment:
```suggestion
    <absolute-convergence-measure limit="1e-1" data="Data2" mesh="MeshOne" />
    <min-iterations value="3" />
```

Reply:
The absolute measure `sufficed` for convergence, so the `min-iteration-convergence` was  completely pointless.
Maybe testing exactly that was the point of the test. I don't know, which why I opened #1846